docs/sql-ref-functions-udf-hive.md

```diff
@@ -52,6 +52,18 @@ SELECT testUDF(value) FROM t;
 |           2.0|
 |           3.0|
 +--------------+
+
+-- Register `UDFSubstr` and use it in Spark SQL.
+-- Note that, it can achieve better performance if the return types and method parameters use Java primitives.
+-- e.g., UDFSubstr. The data processing method is UTF8String <-> Text <-> String. we can avoid UTF8String <-> Text. 
```

Comment:
Example of how to rewrite UDF: 
<img width="1681" alt="image" src="https://user-images.githubusercontent.com/5399861/187021044-ade75cb9-6f3e-40be-aac1-0cd6aab97d0b.png">